src/search/view/SearchRouter.ts

```diff
@@ -9,8 +9,8 @@ export class SearchRouter {
 
 	readonly getRestriction: () => SearchRestriction = memoizedWithHiddenArgument(() => m.route.get(), getRestriction)
 
-	routeTo(query: string, restriction: SearchRestriction, selectedId?: Id | null): void {
-		const { path, params } = getSearchUrl(query, restriction, selectedId ?? undefined)
+	routeTo(query: string, restriction: SearchRestriction, selectedId?: Id | null, selectedEventTime?: ReadonlyArray<number>): void {
```

Comment:
should we maybe turn `selectedId` into `selectedKey` of sorts to not have 2 parameters?